controller/src/main/java/org/jboss/as/controller/transform/SubsystemTransformerRegistration.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2015, Red Hat, Inc., and individual contributors as indicated
```

Comment:
Damn, yeah initial prototype was done than, as I wanted to squeeze it in EAP7 :-)